modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -28,7 +34,7 @@ def type
           message = "No active '#{params[:type]}' intent to file found."
           raise ::Common::Exceptions::ResourceNotFound.new(detail: message) if active_itf.blank?
 
-          render json: ClaimsApi::V2::Blueprints::IntentToFileBlueprint.render(active_itf)
+          render json: ClaimsApi::V2::Blueprints::IntentToFileBlueprint.render(active_itf, root: :data)
```

Comment:
this is causing the `{ data: {} }` - remove the added `, root: :data` for the rswag - my apologies.